src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double firstLeg;
+    private final double secondLeg;
+
+    public RightTriangle(double firstLeg, double secondLeg, String color) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (firstLeg * secondLeg) / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: right triangle, area: " + calculateArea() + " sq"
+                + ", firstLeg " + firstLeg + " units "
+                + ", second leg: " + secondLeg + " units "
+                + " color: " + color);
```

Comment:
In the 'draw()' method, the color of the figure is not specified. It should be 'super.getColor()' instead of just 'color'.

Reply:
I overrode the draw() method in Figure and used getColor() in this method to print the color.